doc/calcurse.1.txt

```diff
@@ -624,10 +624,10 @@ The following structure is created by default in your home directory the
 first time calcurse is run without any options:
 
 ----
-$XDG_DATA_HOME/calcurse/    $XDG_CONFIG_HOME/calcurse/
-          |___apts                    |___conf
-          |___notes/                  |___hooks/
-          |___todo                    |___keys
+$XDG_DATA_HOME/calcurse/        $XDG_CONFIG_HOME/calcurse/
+                  |___apts                          |___conf
+                  |___notes/                        |___hooks/
+                  |___todo                          |___keys
```

Comment:
This looks quite strange, at least in the GitHub source code review. Could you clarify why this is needed/better?

Reply:
It looks like the way it should for me in the GitHub source code review. The reason I added that was because I thought the vertical line should start from `calcurse` instead of the parent directory, like it does in the original one.